### PR TITLE
Fix GzFileStream's atEnd

### PR DESCRIPTION
**Pull request by recloser**

### CODE REVIEW

File: tests/gziptests.nim [hunk 1]
```diff
@@ -43,4 +43,25 @@ proc main() =
                 break
         s.close()
         w.close()
+
+    # read line by line confirming that behavior of atEnd is consistent with standard FileStream
+    block:
+        let fs = newFileStream("files/gzipfiletest.txt")
```
Comment (dom96) [inline tests/gziptests.nim]: 2 space indent is Nim's convention

**Review by dom96 — CHANGES_REQUESTED** [review]

File: tests/gziptests.nim (hunk 1 above)
Comment (recloser) [reply tests/gziptests.nim]: It's consistent with the rest of the test file, though. So do you want me to reindent the entire file or leave it as is?

Comment (dom96) [reply tests/gziptests.nim]: Yes, reindent the whole file please.

Comment (dom96) [reply tests/gziptests.nim]: and gzipfiles too if you can

Comment (recloser) [reply tests/gziptests.nim]: Done.

**Pull request merged**